### PR TITLE
feat: tooltip

### PR DESCRIPTION
**Pull request by Isaacmaamouche**
- Updated the styling
- Added the fade/slide-in animation with 4 different directions depending on the placement property
- Slide-in disabled in the property `fixed` is used or if placement's value is  `auto-*`

👀 [Visit Preview](https://v5-tooltip-1465--welcome-ui.netlify.app)

### CODE REVIEW

File: packages/Tooltip/styles.ts [hunk 1]
```diff
@@ -2,9 +2,43 @@ import styled, { css, system, th } from '@xstyled/styled-components'
...
+}
+type TransformDirectionKey = keyof typeof transformDirection
+
+export const FadeIn = styled.div<{
+  placement?: PlacementOptions
+  fixed?: TooltipOptions['fixed']
+}>`
+  ${th('tooltips')};
+  ${system};
+  background-color: rgba(33, 33, 33, 0.9);
```
Comment (Vico4) [inline packages/Tooltip/styles.ts:25]: get from theme if possible

**Review by Vico4 — CHANGES_REQUESTED** [review]
on the demo there is an extra line on every tooltip

File: packages/Tooltip/styles.ts (hunk 1 above)
Comment (Leiksa) [reply packages/Tooltip/styles.ts:25]: background-color is given on the Tooltip figma -> https://www.figma.com/file/TC5aahnPVDU1hhBcff037e/Feedback?node-id=2%3A890

**Review by Leiksa — CHANGES_REQUESTED** [review]
Screenshot 2022-08-30 at 12 05 32

You seems to have a padding problem or something else. (cf screen)

**Review by Vico4 — APPROVED** [review]
🔥

**Review by Leiksa — APPROVED** [review]

**Review by theo-mesnil — APPROVED** [review]

File: docs/pages/components/tooltip.mdx [hunk 2]
```diff
@@ -46,6 +46,109 @@ Set a `max-width` for long text
 </Tooltip>
 ```
 
+## With a fixed position
+
+Using the `fixed` prop in combination with `placement` will enable a translate animation on your tooltip. The direction works with these values for `placement`.
+
+```jsx
+<Flex gap="lg" flexWrap='wrap'>
+
```
Comment (theo-mesnil) [inline docs/pages/components/tooltip.mdx:55]: Remove all useless break line and align code (like on vscode ;))

**Review by theo-mesnil — APPROVED** [review]

**Pull request merged**